### PR TITLE
button component added

### PR DESCRIPTION
**Pull request by MonikaJhingan**
Please review urgently!!!

### CODE REVIEW

File: Component/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+.btn {
+  height: 5rem;
+  width: 10rem;
+  font-size: 1.8rem;
+  margin: 2rem;
+  padding: 4px 6px;
+  border: none;
+  border-radius: 1rem;
+  color: #fafafa;
+  background-color: none;
```
Comment (abhijeetekad) [inline Component/Button/button.css]: Use colour variable here.

File: Component/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+  color: #fafafa;
+}
+
+.buttons-container {
+  height: 5rem;
+  width: 5rem;
+  border-radius: 50%;
+  background-color: var(--btn-primary-color);
+  box-shadow: 5px -19px 139px -13px rgba(0, 0, 0, 0.75);
+  position: relative;
```
Comment (abhijeetekad) [inline Component/Button/button.css]: Same.. you can use variable

**Review by abhijeetekad — APPROVED** [review]

File: Component/Button/button.css (hunk 1 above)
Comment (MonikaJhingan) [reply Component/Button/button.css]: done

File: Component/Button/button.css [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+  border: none;
+  border-radius: 1rem;
+  color:var(--btn-primary-color
+  background-color: none;
+}
+
+.btn:hover,
+.btn-link:hover {
+  filter: brightness(0.9);
+  cursor: pointer;
```
Comment (Snigdha17) [inline Component/Button/button.css]: it'd be better to put the base button properties like cursor:pointer inside the .btn class, and only have hover effects in hover pseudo-classes

File: Component/Button/button.css [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+.btn {
+  height: 5rem;
+  width: 10rem;
+  font-size: 1.8rem;
+  margin: 2rem;
+  padding: 4px 6px;
+  border: none;
+  border-radius: 1rem;
+  color:var(--btn-primary-color
```
Comment (Snigdha17) [inline Component/Button/button.css]: The bracket for var seems to not have been closed - is the button color applying correctly?

File: Component/Button/button.css [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+  display: inline-block;
+  text-decoration: none;
+  text-align: center;
+  margin: 2rem;
+  padding: 15px 10px;
+  font-size: 1.8rem;
+  font-weight: 300;
+  border: none;
+  border-radius: 1rem;
+  color: #fafafa;
```
Comment (Snigdha17) [inline Component/Button/button.css]: It might be beneficial to create variable for this color, since it's been used in multiple places

**Review by Snigdha17 — COMMENTED** [review]
The code looks good to me, except for some minor comments. It would be really helpful if you could attach deploy preview links for your pull requests so we can see your beautiful work on the screen too :)

File: Component/Button/button.css [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+.outline-default,
+.outline-link {
+  background-color: #fafafa;
+}
+
+.btn-icon {
+  height: 5rem;
+  width: 10rem;
+  font-size: 1.8rem;
+  padding: 4px 6px;
```
Comment (poojavishnoi) [inline Component/Button/button.css]: Try to use be consistent in using rem.

File: Component/Button/button.html [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+        <br>
+          </div>
+      
+        </div>
+        
+        <div class="margin-m1 container-flex flex-center">
+          <code class="code-container">
+            <iframe
+            src="https://carbon.now.sh/embed/YBHU4Nk79OGrq0Uea4XK"
+            style="width: 650px; height: 200px; border:0; transform: scale(1); overflow:hidden;margin-top: 200px;margin-bottom: 20px;"
```
Comment (poojavishnoi) [inline Component/Button/button.html:144]: Try not to use inline css

File: Component/Button/button.css (hunk 3 above)
Comment (MonikaJhingan) [reply Component/Button/button.css]: done

**Review by poojavishnoi — APPROVED** [review]
Code looks fine. I have dropped some suggestion

File: Component/Button/button.css (hunk 4 above)
Comment (MonikaJhingan) [reply Component/Button/button.css]: done

File: Component/Button/button.css (hunk 5 above)
Comment (MonikaJhingan) [reply Component/Button/button.css]: done

File: Component/Button/button.css (hunk 6 above)
Comment (MonikaJhingan) [reply Component/Button/button.css]: done thnx for the suggestions.

File: Component/Button/button.html (hunk 7 above)
Comment (MonikaJhingan) [reply Component/Button/button.html:144]: Sure will keep in mind.

**Pull request merged**